docs/models/using.md

```diff
@@ -15,6 +15,7 @@ Blocks are not directly linked to models, instead, block<em>states</em> are mapp
 5. Get the name of each value with `IProperty<T>#getName(T)`.
```

Comment:
oh, didn't see that. probably would be better to put it in line here. on item 5. or 6.